docs/readthedocs/source/doc/UserGuide/known_issues.md

```diff
@@ -0,0 +1,41 @@
+# BigDL Known Issues
+
+## Spark Dynamic Allocation
+
+By design, BigDL does not support Spark Dynamic Allocation mode, and needs to allocate fixed resources for deep learning model training. But if your environment has already configured Spark Dynamic Allocation, or stipulated that Spark Dynamic Allocation must be used, you may encounter the following error:
+
+> **requirement failed: Engine.init: spark.dynamicAllocation.maxExecutors and spark.dynamicAllocation.minExecutors must be identical in dynamic allocation for BigDL**
+> 
+
+If so, you can set up the `SparkConf` as a workaround.
```

Comment:
Change to `Here we provide a workaround for running BigDL under Spark Dynamic Allocation mode.`